readme.md

```diff
@@ -1,14 +1,16 @@
 # jpegoptim-bin [![Build Status](http://img.shields.io/travis/imagemin/jpegoptim-bin.svg?style=flat)](http://travis-ci.org/imagemin/jpegoptim-bin)
 
-> jpegoptim is a utility for optimizing JPEG files that provides lossless optimization (based on optimizing the Huffman tables) and "lossy" optimization based on setting a maximum quality factor
+> [jpegoptim](https://github.com/tjko/jpegoptim) is a utility for optimizing JPEG files that provides lossless optimization (based on optimizing the Huffman tables) and "lossy" optimization based on setting a maximum quality factor
 
 
 ## Install
 
-```
+```sh
```

Comment:
Don't add `sh` highlight helper. Github's shell syntax highlight is not sophisticated.

For example:

```sh
npm i read-json-sync
```

`read` is unexpectedly highlighted.